File: Go/Easy/1351.Count_Negative_Numbers_in_a_Sorted_Matrix/1351.Count_Negative_Numbers_in_a_Sorted_Matrix.go
package countnegativenumbersinasortedmatrix

func countNegatives(grid [][]int) int {
    if len(grid) == 0 {
        return 0
    }
    c := 0
    for i := 0; i < len(grid); i++ {
        for j := 0; j < len(grid[i]); j++ {
            if grid[i][j] < 0 {
                c++
            }
        }
    }
    return c
}

File: Typescript/1351.Count_Negative_Numbers_in_a_Sorted_Matrix/1351.Count_Negative_Numbers_in_a_Sorted_Matrix.ts
function countNegatives(grid: number[][]): number {
  let c: number = 0;
  for (let i = 0; i < grid.length; i++) {
    for (let j = 0; j < grid[i].length; j++) {
      if (grid[i][j] < 0) {
        c++;
      }
    }
  }
  return c;
}
